config.json.example

```diff
@@ -8,6 +8,7 @@
     "discordReportChannelWebhookURL": null, //URL from discord if you would like notifications when someone makes a report [optional]
     "discordFirstTimeSubmissionsWebhookURL": null, //URL from discord if you would like notifications when someone makes a first time submission [optional]
     "discordCompletelyIncorrectReportWebhookURL": null, //URL from discord if you would like notifications when someone reports a submission as completely incorrect [optional]
+    "neuralBlockURL": "https://ai.neuralblock.app",
```

Comment:
should probably not be defined in the example config, and not used if the app is in development mode.

also if that's the case, include documentation in the readme for setting up an NB instance